sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/implementation/ContainerRegistriesImpl.java

```diff
@@ -68,7 +68,7 @@
         Mono<ContainerRegistriesListRepositoriesResponse> listRepositories(
                 @HostParam("endpoint") String endpoint,
                 @QueryParam("last") String last,
-                @QueryParam("pageSize") Integer pageSize,
+                @QueryParam("n") Integer n,
```

Comment:
Was this a codegen or manual change?

Reply:
Codegen - changed the swagger.

---
In reply to: [602632611](https://github.com/Azure/azure-sdk-for-java/pull/20139#discussion_r602632611) [](ancestors = 602632611)